src/main/java/org/casbin/jcasbin/model/Model.java

```diff
@@ -29,7 +29,7 @@
     private static Map<String, String> sectionNameMap;
 
     static {
-        sectionNameMap = new HashMap<>();
+        sectionNameMap = new LinkedHashMap<>();
```

Comment:
I tend not to use ``LinkedHashMap``. You can just hard-code the order of p, e, r, m. ``LinkedHashMap`` is not very necessary.

Reply:
HashMap doesn't preserve the order of insertion.

Reply:
You can assume the order to be the classic order in Casbin model. It's not necessary to keep exactly the same order as input file.

Reply:
done